website/content/en/preview/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -77,7 +77,15 @@ The following cluster configuration will:
 * Create a role to allow spot instances.
 * Run Helm to install Karpenter
 
-{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-karpenter/scripts/step02-create-cluster.sh" language="bash"%}}
+{{< tabpane text=true right=false >}}
+  {{% tab header="**Create cluster command**:" disabled=true /%}}
+  {{% tab header="With NodeGroup" %}}
```

Comment:
```suggestion
  {{% tab header="Managed NodeGroups" %}}
```

Reply:
Could probably just drop the "With" portion of it and just go with the noun on its own. Apply this across all docs versions if you decide to take this in

Reply:
Changed.